box/info.go

```diff
@@ -0,0 +1,48 @@
+package box
+
+import "github.com/tarantool/go-tarantool/v2"
+
+// ClusterInfo represents information about the cluster.
+// It contains the unique identifier (UUID) of the cluster.
+type ClusterInfo struct {
+	UUID string `msgpack:"uuid"`
+}
+
+// Info represents detailed information about the Tarantool instance.
+// It includes version, node ID, read-only status, process ID, cluster information, and more.
+type Info struct {
+	// The Version of the Tarantool instance.
+	Version string `msgpack:"version"`
+	// The node ID (nullable).
+	ID *int `msgpack:"id"`
+	// Read-only (RO) status of the instance.
+	RO bool `msgpack:"ro"`
+	// UUID - Unique identifier of the instance.
+	UUID string `msgpack:"uuid"`
+	// Process ID of the instance.
+	PID int `msgpack:"pid"`
+	// Status - Current status of the instance (e.g., running, unconfigured).
+	Status string `msgpack:"status"`
+	// LSN - Log sequence number of the instance.
+	LSN uint64 `msgpack:"lsn"`
+	// Cluster information, including cluster UUID.
+	Cluster ClusterInfo `msgpack:"cluster"`
```

Comment:
In the Tarantool 3 the field has name `replicaset`.

Reply:
there should be a custom decoder for this type or extend the type to avoid problems, this implementation is rather minimal

Reply:
Well, that is, I propose in this case, for example, to remove this field altogether, leaving a minimal non-conflicting implementation